NEWS.md

```diff
@@ -1,11 +1,9 @@
-Major changes between releases                  Automated Testing Framework
-===========================================================================
+# Major changes between releases
```

Comment:
This change could be split off/is ready to go!

Reply:
Can you be more explicit about "split off"?

Reply:
Sure: what I was asking for was NEWS.md to have its own branch/PR. I'm ok with the focused set of changes in this file.